usaspending_api/api_contracts/contracts/v2/agency/toptier_code/recipient_award_stats.md

```diff
@@ -0,0 +1,42 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Federal Obligations [/api/v2/agency/{toptier_code}/recipient_award_stats/{?fiscal_year}]
+
+This endpoint is used to power USAspending.gov's agency profile pages.
+
+## GET
+
+This endpoint returns a set of award amount statistics received by a toptier agency in a fiscal year: greatest single award amount, least award amount (note negative amounts are possible), and amounts of awards at median and 25th & 75th percentile.
```

Comment:
This is incorrect, the amounts being returned are the sum award amounts per recipient for the given agency and fiscal year, not the individual award amounts.

Reply:
Also we are restricting the data set so that we are excluding recipients with net negative obligations so negative amounts are not actually possible